spec/defines/validate_db_connection_spec.rb

```diff
@@ -30,7 +30,10 @@
     it { is_expected.to contain_postgresql__validate_db_connection('test') }
 
     it 'has proper path for validate command' do
-      is_expected.to contain_exec('validate postgres connection for test@test:5432/test').with(unless: %r{^/usr/local/bin/validate_postgresql_connection.sh\s+\d+})
+      is_expected.to contain_exec('validate postgres connection for test@test:5432/test').with(unless: ['/usr/local/bin/validate_postgresql_connection.sh',
+                                                                                                        4,
+                                                                                                        30,
+                                                                                                        "' /usr/bin/psql --tuples-only --quiet  -h test  -U test  -p 5432  --dbname test  '"])
```

Comment:
The quote here looks odd. I'd expect:
```suggestion
                                                                                                        "/usr/bin/psql --tuples-only --quiet  -h test  -U test  -p 5432  --dbname test"])
```

Reply:
I had them added onto the end due to it having been done that way in the original code.
Can try without them and see if it errors though

Reply:
Seems to work fine